### ISSUE TITLE
[Bug] remove your test folder from the package

### ISSUE THREAD

**Issue by ngam** [pytorch/botorch#1019]
# 🐛 Bug

## To reproduce

see this issue for reference: https://github.com/conda-forge/botorch-feedstock/issues/9

** Code snippet to reproduce **
```
conda install botorch gpytorch
```

** Stack trace/error message **
```
// Paste the bad output here!
ClobberWarning: This transaction has incompatible packages due to a shared path.
  packages: conda-forge/noarch::gpytorch-1.5.1-pyhd8ed1ab_0, conda-forge/noarch::botorch-0.2.2-py_0
  path: 'lib/python3.9/site-packages/test/optim/__pycache__/__init__.cpython-39.pyc'

```

## System information

**Please complete the following information:**
- 
- 
- 
- 

## Additional context

You can add `recursive-exclude test *` or something to your `MANIFEST.in`. As a workaround, we have simply force-deleted the tests folder on the conda-forge side for now, pending a fix from you.

**Comment by saitcakmak** [comment]
This is not directly related to the issue, but it is worth noting that the package versions for gpytorch and botorch on condaforge are 1.5.1 and 0.2.2. Gpytorch just released 1.6.0, so that's quite recent, but the 0.2.2 for botorch is quite old. The latest release is 0.5.1 which has been out since Sep 2.

Interestingly, when I run `conda install botorch` I get `botorch            conda-forge/noarch::botorch-0.5.1-pyhd8ed1ab_1`, which is the correct version (gpytorch is still 1.5.1). Just thought it might be worth mentioning.

**Comment by ngam** [comment]
> This is not directly related to the issue, but it is worth noting that the package versions for gpytorch and botorch on condaforge are 1.5.1 and 0.2.2. Gpytorch just released 1.6.0, so that's quite recent, but the 0.2.2 for botorch is quite old. The latest release is 0.5.1 which has been out since Sep 2.

I took this out from the referenced issue. The point is though, both botorch and gpytorch should remove the test folder from root. You can also just download the actual release packages (from both botorch and gpytorch; pypi or conda channels) and you will see the test folder will be there, and that causes a conflict, so it is a better (and simple) practice to just remove it.

**Comment by Balandat** [comment]
Fixed by #1020.
